llvm/test/Verifier/AMDGPU/intrinsic-amdgpu-init-exec-from-input.ll

```diff
@@ -0,0 +1,27 @@
+; RUN: not llvm-as -disable-output 2>&1 %s | FileCheck %s
+
+; Function Attrs: convergent nocallback nofree nounwind willreturn
+declare void @llvm.amdgcn.init.exec.from.input(i32, i32 immarg) #0
+attributes #0 = { convergent nocallback nofree nounwind willreturn }
+
+; CHECK: only inreg arguments to the parent function are valid as inputs to this intrinsic
+; CHECK-NEXT: call void @llvm.amdgcn.init.exec.from.input(i32 0, i32 0)
+define amdgpu_ps void @init_exec_from_input_fail_immarg(i32 inreg %a, i32 %b) {
```

Comment:
Do we really need `amdgpu_ps` for these tests?

Reply:
No, removing it works, too. (It happened to be there because I copied and adapted the situation from the fuzzer testcase.) Should I remove it?

Reply:
I removed it